Lib/test/test_inspect.py

```diff
@@ -4150,13 +4150,14 @@ def func(*args, **kwargs):
 
     def test_base_class_have_text_signature(self):
         # see issue 43118
-        from io import BufferedReader
+        from .ann_module7 import BufferedReader
```

Comment:
```suggestion
        from test.ann_module7 import BufferedReader
```
The current version will likely make running `python LIb/test/test_inspect.py` directly fail